bin/json-refs

```diff
@@ -146,7 +146,7 @@ program
         if (errors.length > 0 && !that.force) {
           throw new Error('Document has invalid references:\n\n' + errors.join('\n'));
         } else {
-          console.log(that.yaml ?
+          console.log(that.yaml || location.split('.').pop() == 'yaml' ?
```

Comment:
What if you just did `location.endsWith('.yaml')`?  That's a little more clear and would have fewer false positives.